framework/src/seclink/seclink.c

```diff
@@ -174,6 +174,48 @@ int sl_deinit(sl_ctx hnd)
 	return 0;
 }
 
+#ifdef CONFIG_ARM_TRUSTZONE
+int sl_allocate(sl_ctx hnd, uint32_t ssize)
+{
+	SL_ENTER;
+	if (!hnd || ((struct _seclink_s_ *)hnd)->fd <= 0) {
+		SL_ERR(-1);
+		return -1;
+	}
+
+	struct _seclink_s_ *sl = (struct _seclink_s_ *)hnd;
+	struct seclink_comm_info info = {.priv = NULL, .s_size = ssize, .s_context = NULL};
+	struct seclink_req req = {.req_type.comm = &info, 0};
+	DiagPrintf("nits alloc\n");
```

Comment:
What is the difference between `DiagPrintf` and `SL_LOG`?

Reply:
this was debug print. I have removed it